public/Day-222/scripts/physics/gravity.js

```
@@ -0,0 +1,463 @@
+
+    computeAcceleration(body, allBodies) {
+        if (this.useQuadTree && allBodies.length > 10) {
+            // Use quad-tree for optimization
+            const boundary = {
+                x: 0,
+                y: 0,
+                width: this.mainCanvas.width,
+                height: this.mainCanvas.height
+            };
+            const tree = new QuadTree(boundary);
+            allBodies.forEach(b => tree.insert(b));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)